program-interview-preparation.html

```diff
@@ -9,18 +9,91 @@ <h1>Interview Preparation</h1>
 <div class="container">
     <div class="content-constrained">
         <div class="section">
-            <h2><a href="#resume-clinic">Resume Clinic</a></h2>
-            <p>TODO</p>
+            <h2>CV Clinic</h2>
+            <p>Are you ready to take your CV to the next level? Join us for an exciting opportunity to elevate your professional profile with our CV Clinic! 🚀</p>
+            <p>At the Women Coding Community, we believe in the power of collaboration and support. That's why we're thrilled to offer our members a unique chance to receive personalized feedback and guidance on their CVs from their peers.</p> 
+            <p>Our CV Clinic is designed to provide you with valuable insights, expert tips, and constructive feedback to help you craft a standout CV that showcases your true potential.</p>
+            <div class="faq my-4">
+                <section id="accordion">
+                    <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-1" aria-expanded="false" data-toggle="collapse" href="#faq-1" role="button">
```

Comment:
maybe not for now, but let's keep in mind to create a new class for this kind of things. The html is really big and hard to maintain. 
```suggestion
                    <a class="prep-link" aria-controls="interview-1" aria-expanded="false" data-toggle="collapse" href="#interview-1" role="button">
```

Reply:
got it, yeah a lot of this is from the old code base. Definitely want to create a lot of reusable style patterns post-launch